### DISCUSSION TITLE
MPD (Music Player Daemon) server support

### DISCUSSION THREAD

**Discussion by Zetten** [audioling/audioling#15] [Ideas]
As a flexible & largely protocol-agnostic media browser and player, it seemed Feishin (and now Audioling) would be a nice solution to the notable lack of beautiful, up-to-date, user-friendly desktop clients for [MPD](https://en.wikipedia.org/wiki/Music_Player_Daemon).

I spent an evening or two looking at this in Feishin and I think basic library functionality should be straightforward (just different API calls), but playback has a couple of interesting options:

* simply control playback on the MPD server - no local mpv/web audio
* stream to a local player using an httpd output - requires prior setup in mpd server config, but allows for a non-local server to playback on the client machine

**Comment by jeffvli** [comment]
I can look into it, but it would be a third priorty after Subsonic and Jellyfin.
